### PR TITLE
MES-2187 Adapting the the new schema

### PR DESCRIPTION
**Pull request by szabolcs-keresztes**
- [x] Adopting the new Vehicle Checks schema
- [x] Moving VehicleChecks state to TestData

### CODE REVIEW

File: src/pages/office/office.ts [hunk 1]
```diff
@@ -196,10 +199,12 @@ export class OfficePage extends BasePageComponent {
         select(getAdditionalInformation),
       ),
       showMeQuestion$: currentTest$.pipe(
+        select(getTestData),
```
Comment (johnmorleybjss) [inline src/pages/office/office.ts]: As we're piping `currentTest$`, why do we need to do `getTestData` again?

File: src/pages/waiting-room-to-car/waiting-room-to-car.ts [hunk 2]
```diff
@@ -178,14 +179,17 @@ export class WaitingRoomToCarPage extends BasePageComponent {
         map(isManual),
       ),
       tellMeQuestionSelected$: currentTest$.pipe(
+        select(getTestData),
```
Comment (johnmorleybjss) [inline src/pages/waiting-room-to-car/waiting-room-to-car.ts]: Same as on Office, we're doing `currentTest$.pipe`, wouldn't have thought that `getTestData` was necessary.

File: src/pages/office/office.ts (hunk 1 above)
Comment (szabolcs-keresztes) [reply src/pages/office/office.ts]: Good point. Weird that it works fine..
Will change it.

Comment (szabolcs-keresztes) [reply src/pages/office/office.ts]: No, you need the `getTestData` selector. The vehicle checks are part of the `testData`

Comment (johnmorleybjss) [reply src/pages/office/office.ts]: Ah yes, I understand now, `testData` is the key within the individual test, fair enough. 😄

**Pull request merged**